beta/experience-platform-extension/set-up-the-sdk.md

```diff
@@ -168,8 +168,11 @@ Parameters:
 * `experienceEvent (required)`should not be null.
 * `responseCallback (optional)`callback is invoked when the response handles are received from Experience Edge. It may be called on a different thread and may be invoked multiple times.
 
+> Note: When the Edge extension receives an `ExperienceEvent`, it is persisted to disk. This means that if an `ExperienceEvent` fails to be sent to the Edge network (usually due to a network connectivity issue), it will be retried at a later date, even between app launches.
```

Comment:
@nporter-adbe this is not supported yet in Android, we should add this comment for iOS only. Otherwise let's document in aepsdk-edge-ios repo